recipes-extended/quark/files/quark.initscript

```diff
@@ -0,0 +1,41 @@
+#! /bin/sh
+
+set -e
+
+QUARK="/usr/bin/quark"
+QUARK_OPTS="-h 1.0.0.0 -p 80 -d /usr/lib/xui"
+PIDFILE="/var/run/quark.pid"
+
+start_quark() {
+    LD_PRELOAD=/usr/lib/libv4v-1.0.so.0.0.0 INET_IS_V4V=1 start-stop-daemon --start --background --make-pidfile --pidfile="${PIDFILE}" --quiet --oknodo --exec $QUARK -- $QUARK_OPTS
+}
+
+stop_quark() {
+    start-stop-daemon --stop --quiet --oknodo --pidfile="${PIDFILE}"
+    rm -f "${PIDFILE}"
+}
+
+test -x $QUARK || exit 0
+
+case "$1" in
+  start)
+    echo "Starting Quark server"
+    start_quark
+    ;;
+  stop)
+    echo "Stopping Quark server"
+    stop_quark
+    ;;
+
```

Comment:
Just a nitpick. Inconsistent whitespacing, some have a blank space between branches and some do not.